controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -162,21 +161,8 @@ func (r *StorageConsumerReconciler) reconcileEnabledPhases() (reconcile.Result,
 			return reconcile.Result{}, err
 		}
 
-		if r.storageConsumer.Status.Client.OperatorVersion == "" {
-			return reconcile.Result{}, fmt.Errorf("client does not expose operator version in its status")
-		}
-
-		clientOperatorVersion, err := semver.Parse(r.storageConsumer.Status.Client.OperatorVersion)
-		if err != nil {
-			return reconcile.Result{}, fmt.Errorf("malformed ClientOperatorVersion: %v", err)
-		}
-		if clientOperatorVersion.Major == 4 &&
-			clientOperatorVersion.Minor == 18 {
-			// TODO (leelavg): need to be removed in 4.20
-			// We have a new controller which maps the resources from 4.18 to 4.19 way of management,
-			// until the resources are mapped we don't want to reconcile consumer belonging to connected client
-			// relax the condition on knowing the resources are mapped in a separate PR
-			return reconcile.Result{}, nil
+		if _, notAdjusted := r.storageConsumer.GetAnnotations()[util.TicketAnnotation]; notAdjusted {
+			return reconcile.Result{}, fmt.Errorf("StorageConsumer has not been adjusted as per 4.19")
```

Comment:
Can we have a better message here. One that indicate that this is an upgraded 4.18?